addons/payment/static/src/js/payment_processing.js

```diff
@@ -81,14 +81,16 @@ odoo.define('payment.processing', function (require) {
                 'tx_cancel': [],
                 'tx_error': [],
             };
+
+            if (transactions.length > 0 && transactions[0].acquirer_provider == 'transfer') {
```

Comment:
We should pay attention at the state of the transaction, we don't want to redirect to the return url if it is in an error state.
Plus what would happen if we have two pending orders?